e2e/nomostest/artifactregistry/image.go

```diff
@@ -51,6 +50,11 @@ func SetupImage(nt *nomostest.NT, name, version string) (*Image, error) {
 	if version == "" {
```

Comment:
Can you move this empty check into `validateImageVersion` so it is also checked when SetVersion is called?

Reply:
moved